### DISCUSSION TITLE
Note for FreeBSD - Compile with FPC

### DISCUSSION THREAD

**Discussion by galfar** [galfar/deskew#64] [Q&A]
**[Original report](https://bitbucket.org/galfar/app-deskew/issue/13) by Anonymous.**

----------------------------------------

The RTL includes (which encompass StrUtil) are not included in the fpc base install on FreeBSD (I'm on 11.1-RELEASE).

If anyone gets here from google due to...

```
CmdLineOptions.pas(36,3) Fatal: Can't find unit StrUtils used by CmdLineOptions
```

Then you need to install the fpc-rtl packages...

```
#> pkg search fpc-rtl
fpc-rtl-console-3.0.4          Free Pascal console abstraction unit (RTL)
fpc-rtl-extra-3.0.4            Free Pascal rtl-extra unit
fpc-rtl-objpas-3.0.4_1         Free Pascal compatibility unit with Delphi
fpc-rtl-unicode-3.0.4          Free Pascal compatibility unit with Delphi
```

I'm honestly not sure which one StrUtils is in, I just installed them all.

**Comment by polluks** [comment]
I have the same issue with NetBSD
/usr/local/lib/fpc/3.2.2/units/x86_64-netbsd/rtl-objpas/strutils.ppu
